<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CodecComparisonWindow</class>
 <widget class="QWidget" name="CodecComparisonWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1443</width>
    <height>1031</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="outerLayout" rowstretch="1,1" columnstretch="1,1">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_0" stretch="1,10,0">
       <item>
        <widget class="QLabel" name="original">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>10</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="VlcWidgetVideo" name="rawVideo" native="true">
         <property name="styleSheet">
          <string notr="true">Background-color: #000;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true"/>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10,3,3">
       <item>
        <widget class="QLabel" name="first">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>10</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="VlcWidgetVideo" name="encodedVideo1" native="true">
         <property name="styleSheet">
          <string notr="true">Background-color: #000;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="VideoStatisticsWidget" name="videoInfo1" native="true"/>
       </item>
       <item>
        <widget class="CodecParametersWidget" name="encodedParams1" native="true"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,10,3,3">
       <item>
        <widget class="QLabel" name="second">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>10</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="VlcWidgetVideo" name="encodedVideo2" native="true">
         <property name="styleSheet">
          <string notr="true">Background-color: #000;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="VideoStatisticsWidget" name="videoInfo2" native="true"/>
       </item>
       <item>
        <widget class="CodecParametersWidget" name="encodedParams2" native="true"/>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,10,3,3">
       <item>
        <widget class="QLabel" name="third">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>10</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="VlcWidgetVideo" name="encodedVideo3" native="true">
         <property name="styleSheet">
          <string notr="true">Background-color: #000;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="VideoStatisticsWidget" name="videoInfo3" native="true"/>
       </item>
       <item>
        <widget class="CodecParametersWidget" name="encodedParams3" native="true"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VlcWidgetVideo</class>
   <extends>QWidget</extends>
   <header location="global">VLCQtWidgets/WidgetVideo.h</header>
   <container>1</container>
   <slots>
    <signal>fullscreen()</signal>
    <slot>toggleFullscreen()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>VideoStatisticsWidget</class>
   <extends>QWidget</extends>
   <header>VideoStatisticsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CodecParametersWidget</class>
   <extends>QWidget</extends>
   <header location="global">CodecParametersWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
